### DISCUSSION TITLE
Is it possible to implement this kind of custom annotation?

### DISCUSSION THREAD

**Discussion by yaorx163** [notnotmelon/factorissimo-2-notnotmelon#80] [Ideas]
Is it possible to implement this kind of custom annotation?
![Snipaste_2024-11-03_21-20-32](https://github.com/user-attachments/assets/6d7f40e5-fd0b-473f-aab7-35b7a61c81d9)

**Comment by notnotmelon** [comment]
This would be very cool. It's also extremely hard to implement correctly and will likely take a week of my time. I will not do this.
For now I recommend to use the preview camera feature.
